src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -33,7 +38,17 @@
     "Mongo",  # Node
     "Postgres",  # Node
 ]
+
 PARAMETERIZED_SQL_QUERY_REGEX = re.compile(r"\?|\$1|%s")
+CONTAINS_PARAMETER_REGEX = re.compile(
+    join_regexes(
+        [
+            r"'(?:[^']|'')*?(?:\\'.*|'(?!'))",  # single-quoted strings
+            r"\b(?:true|false)\b",  # booleans
+            r"\?|\$1",  # existing parameters
```

Comment:
I'll get in touch with @mjq-sentry 

I'll add the %s into it, but I think just checking for ? , $1 and %s should be enough. No need for $2, 3 etc, because we just care about if there is a paramater, not how many there are are.